SkyBlock/README.md

```diff
@@ -0,0 +1,358 @@
+8. `/is team promote` is used to promote a player up a rank.
+9. `/is team reject` is used to reject a team invite.
+10. `/is team setowner` is used to transfer ownership of the island to another player.
+11. `/is team uncoop` is used to remove the coop role from a player.
+12. `/is team untrust` is used to untrust a player on your island.
+
+Learn more about teams in [Teams](#Teams).
+
+### `top`
+
+`/is top` is used to show the top ten islands on TrixMC SkyBlock.
+
+### `warp`
+
+`/is warp <player>` is used to warp to a player's island.
+
+### `warps`
+
+`/is warps` is used to list all available warps on TrixMC SkyBlock. More info on warps can be found in [Warps](#Warps-1).
+
+### `bank`
+
+`/is bank deposit <amount>` is used to deposit money into your island bank from your wallet.
+`/is bank withdraw <amount>` is used to withdraw money into your island bank from your wallet.
+`/is bank balance` is used to view your island bank balance.
+`/is bank statement` is used to view your island bank statement.
+
+More info on money and bank can be found in [Money](#Money).
+
+### `balance`
+
+`/balance` is used to check your wallet balance.
+
+More info on money and bank can be found in [Money](#Money).
+
+### `chat`
+
+`/is chat` is used to toggle island chat on and off. This allows you to communicate with ONLY your teammates.
+
+### `teamchat`
+
+`/is teamchat` is used to toggle team chat. This allows you to communicate with ONLY those in your team.
+
+## Island Types
+
+There are many different types of islands you can choose from while creating an island on TrixMC SkyBlock.
+
+### List of island Types
+
+A list of island types shall be added in the near future.
+
+## Money
+
+As you progress, you will need to use money and make transactions. Money is stored in one of two places: wallet or bank. You can see the balance of both on the right hand side of your screen.
+
+### Interest
+
+For money stored in your bank, you recieve an interest. The interest is credited every hour you are online. The rate of interest is not revealed and keeps changing.
+
+## Challenges
+
+Challenges let you complete various tasks and recieve rewards. These rewards are in the form of items and experience. These items are useful in your journey to beat the gamemode. You can check your challenges using the command [`/is challenges`](#challenges).
+
+### List of Challenges
+
+This list shall be added in the near future.
+
+## Settings
+
+Settings allow you to customize your island. You can view and change your settings using the command [`/is settings`](#settings).
+
+There are currently 59 Protection Settings, 6 General Settings and 17 SkyBlock Island Settings that can be applied to the SkyBlock Island. Each of which ENABLE or DISABLE access to one or more types of players: Visitor, Coop, Trusted, Member, Sub-Owner and Owners for that island.
+
+### List of settings
+
+<!-- The below table was copy pasted from the BentoBox wiki -->
```

Comment:
Don't use HTML in markdown. please.

Reply:
HTML tables are easier and more intuitive than md.